apps/frontend/src/app/3_organisms/BorrowLoan/components/ExtendLoanForm/ExtendLoanForm.tsx

```diff
@@ -0,0 +1,394 @@
+          label={t(pageTranslations.labels.nextRolloverDate)}
+        />
+
+        <div className="flex md:flex-row flex-col mb-3">
+          <Paragraph className="font-medium text-xs mb-3 text-gray-30 md:mr-8">
+            {t(pageTranslations.labels.payPrepaidInterest)}
+          </Paragraph>
+
+          <div className="flex items-center">
+            <RadioButton
+              label={t(pageTranslations.labels.collateral)}
+              id="useCollateral"
+              className="mr-6"
+              checked={useCollateral}
+              onChange={e => setUseCollateral(e.target.checked)}
+            />
+            <RadioButton
+              label={t(pageTranslations.labels.wallet)}
+              id="useWallet"
+              checked={!useCollateral}
+              onChange={e => setUseCollateral(!e.target.checked)}
+            />
+          </div>
+        </div>
+
+        {useCollateral && (
+          <>
+            <div className="flex items-center mb-8 gap-3">
+              <AmountInput
+                value={collateralChange.toString()}
+                label={t(translations.common.amount)}
+                className="w-full flex-grow-0 flex-shrink"
+                placeholder="0"
+                readOnly
+              />
+              <div className="bg-gray-60 p-2.5 min-w-[5.5rem] rounded">
+                <AssetRenderer asset={collateralToken} showAssetLogo />
+              </div>
+            </div>
+
+            <SimpleTable>
+              <SimpleTableRow
+                label={t(pageTranslations.labels.newRolloverDate)}
+                value={renderNewRolloverDate}
+                valueClassName="text-primary-10"
+              />
+              <SimpleTableRow
+                label={t(pageTranslations.labels.newCollateralBalance)}
+                value={
+                  <DynamicValue
+                    initialValue="0"
+                    value={newCollateralAmount.toString()}
+                    renderer={() =>
+                      renderValue(
+                        newCollateralAmount.toString(),
+                        collateralToken,
+                      )
+                    }
+                  />
+                }
+              />
+            </SimpleTable>
+
+            <div className="flex flex-row justify-between items-center mt-6 mb-3">
+              <div className="flex flex-row justify-start items-center gap-2">
+                <span>
+                  {t(translations.adjustCreditLine.labels.collateralRatio)}
+                </span>
+              </div>
+              <div className="text-primary-10">
+                <DynamicValue
+                  initialValue="0"
+                  value={collateralRatio.toString()}
+                  renderer={value => renderValue(value, '%', 0)}
+                />
+              </div>
+            </div>
+
+            <HealthBar
+              start={COLLATERAL_RATIO_THRESHOLDS.START}
```

Comment:
`COLLATERAL_RATIO_THRESHOLDS` is not valid in this case - first, validate with @john-light / @ororopickpocket if you need to use `initial collateral ratio` or `min collateral ratio`. I think `initial collateral ratio` is the correct one in this case and if so, look at `collateralRatioThresholds` in `AdjustLoanForm` and use that.

Reply:
@tiltom you are correct